cdap-examples/WordCount/src/main/java/co/cask/cdap/examples/wordcount/WordCounter.java

```diff
@@ -41,15 +41,21 @@
  */
 public class WordCounter extends AbstractFlow {
 
+  private final WordCount.WordCountConfig config;
+
+  public WordCounter(WordCount.WordCountConfig config) {
+    this.config = config;
```

Comment:
Oh, so this is used only in Flow. And Flow's methods are executed only during configuration time anyway. 

Property annotation is used if we want things to be available in runtime. So Flowlets, HttpServiceHandlers use them.